lib/generators/lp_token_auth/templates/initializer.rb.erb

```diff
@@ -19,4 +19,14 @@ LpTokenAuth.config do |config|
   # default: [:cookie]
   #
   config.token_transport = [:cookie]
+
+  # Where to find the required JWE_PRIVATE_KEY value
```

Comment:
stylistic question (I see this is using the same pattern as above) but in other gems I often see the default expressed inline as a commented out line:

```
# Where to find the required JWE_PRIVATE_KEY value
# config.jwe_private_key = ENV['JWE_PRIVATE_KEY']
```

not sure if there's an agreed upon majority convention (this is just an observation, not a request to change)